stix_shifter_modules/msatp/configuration/config.json

```diff
@@ -37,6 +37,10 @@
             },
             "clientSecret": {
                 "type": "password"
+            },
+            "access_token": {
```

Comment:
why did you add `access_token` in the configuration?

Reply:
The way the original connector works is when you set the tenant, clientId and clientSecret, it will generate an access token and place it in the configuration under `access_token` field automatically. If you don't have this field in the config, when the config is being validated after changed by the connector it will raise an error that the structure is incorrect, thus I added this field as optional in order to bypass this structure change validation error.

Reply:
Yeah we are aware of this issue. this is a bug in cp4s 1.6 and earlier. It is fixed in 1.7. so you don't need to add `access_token` as optional anymore

Reply:
@mdazam1942 ok I removed it